VK/VK/Services/NetworkService.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  NetworkService.swift
+//  VK
+//
+//  Created by Дмитрий Супрун on 22.11.22.
```

Comment:
```suggestion
//  NetworkService.swift
//  VK
//
//  Created by Дмитрий Супрун on 22.11.22.
```
форматтер шапку не пишет?